src/extension/installhelper.js

```diff
@@ -46,15 +43,15 @@
       }
 
       const { owner, repo } = getGitHubSettings(giturl);
-      const configIndex = projects.findIndex((cfg) => cfg.owner === owner && cfg.repo === repo);
-      if (configIndex < 0 && owner && repo) {
+      const project = projects.find((cfg) => cfg.owner === owner && cfg.repo === repo);
+      if (project) {
         log.info('installhelper.js: project not added yet');
         if (addProjectContainer) {
           // instrument add project button
           const button = addProjectContainer.querySelector('a');
           if (button.dataset.sidekickExtension !== giturl) {
             button.onclick = () => {
-              addProject({ giturl, project }, () => window.location.reload());
+              chrome.runtime.sendMessage({ action: 'addRemoveProject' });
```

Comment:
is `window.location.reload()` not needed anymore? otherwise you could wait for a reply from the worker.

Reply:
No, this is already handled in the `addRemoveProject` action.